plugins/backfill/main.py

```diff
@@ -176,3 +162,40 @@ def background(self):
     def history(self):
         """ Outputs recent user request history """
         return flask.Response(file_ops('r'), mimetype='text/txt')
+
+
+def generate_backfill_command(dag_name: str,
```

Comment:
i'm confused as to why you refactored this to use a helper function generate_backfill_command. Originally I thought it was because you wanted to re-use the same function for the backfill dag, but I see that this function is also defined in dags/utils/backfill.

Reply:
So it's testable. I use the helper function in `test_generate_backfill_command` to make sure we match the `BackfillParams` method.

Reply:
Actually, importing the helper function in a test run has the side effect of loading a bunch of flask stuff. Behaviour is inconsistent from local to CI. Now that I've validated the outputs to be the same between the refactored version and the old version, I'll switch to the new version in the backfill plugin.